auto_round/autoround.py

```diff
@@ -217,6 +216,7 @@ def __init__(
         disable_deterministic_algorithms = kwargs.pop("disable_deterministic_algorithms", False)
         static_kv_dtype = kwargs.pop("static_kv_dtype", None)
         self.vlm = kwargs.pop("vlm") if "vlm" in kwargs else False
+        self.mem_expansion_factor = kwargs.pop("mem_expansion_factor", None)
```

Comment:
ram_per_param_scale? and better have a comment to show the meaning of this variable